packages/components/src/segmented-control/component.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+// eslint-disable-next-line no-restricted-imports
+import { RadioGroup, useRadioState } from 'reakit';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useRef } from '@wordpress/element';
+import { useMergeRefs, useResizeObserver } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { contextConnect, useContextSystem } from '../ui/context';
+import { View } from '../view';
+import * as styles from './styles';
+import { useUpdateEffect } from '../utils/hooks';
+import Backdrop from './segmented-control-backdrop';
+import Button from './segmented-control-button';
+import type { SegmentedControlProps } from './types';
+
+function SegmentControl( props: any, forwardedRef: any ) {
+	const {
+		className,
+		baseId,
+		isAdaptiveWidth = false,
+		isBlock = false,
+		id,
+		label = __( 'SegmentControl' ),
```

Comment:
This default value is pretty useless. `label` should be a required prop instead of having a bad default like this (also has the added benefit of reducing the usage of `i18n` in the components package.